Install files before trying to update the icon cache or compile the schemas.
https://github.com/virt-manager/virt-manager/issues/400
--- setup.py.orig	2022-02-12 10:13:59.000000000 -0600
+++ setup.py	2022-06-03 14:36:29.000000000 -0500
@@ -264,6 +264,8 @@
         setuptools.command.install.install.finalize_options(self)
 
     def run(self):
+        setuptools.command.install.install.run(self)
+
         if not self.distribution.no_update_icon_cache:
             print("running gtk-update-icon-cache")
             icon_path = os.path.join(self.install_data, "share/icons/hicolor")
@@ -275,8 +277,6 @@
                 "share/glib-2.0/schemas")
             self.spawn(["glib-compile-schemas", gschema_install])
 
-        setuptools.command.install.install.run(self)
-
 
 ###################
 # Custom commands #
